apps/wallet/src/background/keyring/AccountKeypair.ts

```diff
@@ -15,9 +16,10 @@ export class AccountKeypair {
     }
 
     async sign(data: Uint8Array): Promise<SerializedSignature> {
+        const digest = blake2b(data, { dkLen: 32 });
```

Comment:
why `32`?

Reply:
@williamrobertson13 good question, we use blake2b with 32byte (256bits) outputs

Reply:
thanks for the explainer! not required for this diff, but I wonder if we should codify this better at the SDK level with a constant, e.g. `TRANSACTION_DIGEST_LENGTH`